docs/docs/addons-perf.md

```diff
@@ -57,6 +62,34 @@ The following methods use the measurements returned by [`Perf.getLastMeasurement
 
 * * *
 
+## Example Usage
+
+We will take simple example of a form which can be used to submit comments and then listing all comments. The example code is [available on GitHub](https://github.com/dhyey35/react-example-for-performance/blob/master/public/scripts/example.js).
+
+```javascript
+import { Component } from 'react'
+import Perf from 'react-addons-perf'
+
+const Comment = (props) => (
+	<div className="comment">
+	  <h2 className="commentAuthor">
+	    {props.author}
+	  </h2>
+	  <span>{props.children.toString()}</span>
+	</div>
+ )
+//code...
+  
+ReactDOM.render(
+  <CommentBox />,
+  document.getElementById('content')
+);
+```
+Open the developer console in the browser and type `Perf.start()`. Then perform the action you want to monitor, like submitting a form. Finally, type `Perf.stop()` and `Perf.getLastMeasurements()` to get the measurements. See the Reference below for more methods.
```

Comment:
`Perf.getLastMeasurements()` is an extremely low level method and gives information that has very little value to the user unless they know how to interpret it. This is definitely *not* what we should be suggesting to the user as an introductory example.